lib/neo4j/active_node.rb

```diff
@@ -57,6 +57,18 @@ def neo4j_obj
       _persisted_obj || fail('Tried to access native neo4j object on a non persisted object')
     end
 
+    def method_missing(method, *params, &block)
+      if self.class.full_scopes.keys.include? method
+        self.as(:a).public_send(method, *params, &block)
```

Comment:
I am not sure if hardcoded alias `:a` is the right way to go. There are already some naming alias naming rules in query and association proxy. We probably should tap into that.